core/hw/pvr/Renderer_if.cpp

```diff
@@ -229,6 +229,8 @@ bool rend_single_frame()
 	return do_swp;
 }
 
+int rend_en = true;
```

Comment:
I would like to have a comment here or a more meaningful name (what does the ``_en`` mean?).
Maybe we should call it ``rend_end`` or ``rend_done`` and use ``while (!rend_end/done)`` below?

Reply:
this stands for rendering enabled, iirc it's the control flag of the rendering thread?